decidim-core/spec/content_renderers/decidim/link_renderer_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  describe ContentRenderers::LinkRenderer do
+    let(:renderer) { described_class.new(content) }
+    let(:urls) do
+      %w(
+        http://example.com/SystemTestHtmlScreenshotsurl
+        https://example.com/url
+        http://localhost:3000/some/url
+        http://example.com/引き割り.html
+        http://example.com/%E5%BC%95%E3%81%8D%E5%89%B2%E3%82%8A.html
+        https://nörttilive.fi
+        http://www.sinara.technology/about/
```

Comment:
Maybe we shouldn't advertise any particular services/companies here, so these could be changed:
- https://nörttilive.fi => https://ëxämplö.com (testing special characters used in Europe within the domain)
- http://www.sinara.technology/about/ => http://www.example.technology/about/ (testing unusual TLD)